modules/security-network-namespaces.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-namespaces_{context}"]
+= Using network namespaces
+
+{product-title} uses software-defined networking (SDN) to provide a unified
+cluster network that enables communication between containers across the
+cluster.
+
+Using network namespaces, you can isolate pod networks. Each pod gets its own IP
+and port range to bind to, thereby isolating pod networks from each other on the
+node. Pods from different projects cannot send packets to or receive packets
```

Comment:
This is not correct. In network policy mode, by default, all Pods in a project are accessible from other Pods and network endpoints, and pods become isolated by having a NetworkPolicy that selects them. In multitenant  mode, it provides project-level isolation for pods and services.

Reply:
Okay. I reworded that paragraph as you suggested.